bin/setenv.sh.template

```diff
@@ -4,3 +4,6 @@ export SLACK_CLIENT_ID=
 export SLACK_CLIENT_SECRET=
 export ROOT_URL=
 export MONGODB_URI=
+
+export DEFAULT_ADMIN_USERNAME=
```

Comment:
Please avoid using environment variable when not needed. 
In this case, we can assume that at the installation of the app, the default user/password is admin/admin. When logged in, the admin creates another user and deletes the admin account. 
Just write it down it in the install doc.
In the future we'll force the admin user to change its password at his first login.

Reply:
Ok, I dit it because it was the only way for me not to write the default username/pw in the code.
But np if the user is told that he needs to create directly a user.
I'll fix it, and write direclty the default credentials in the code.